libs/sharepoint_connector.py

```
@@ -0,0 +1,24 @@
+from office365.runtime.auth.user_credential import UserCredential
+from office365.sharepoint.client_context import ClientContext
+from office365.sharepoint.files.file import File
```

Ruff (0.8.2)

`office365.sharepoint.files.file.File` imported but unused

Remove unused import: `office365.sharepoint.files.file.File`

(F401)